lighthouse-core/test/results/sample_v2.json

```diff
@@ -3241,15 +3241,15 @@
           },
           {
             "name": "http://localhost:10200/dobetterweb/empty_module.js?delay=500",
-            "resourceBytes": 60
+            "resourceBytes": 0
```

Comment:
ok so this is where it matters: when a scriptelement references an invalid script resource, we still try to show it in the treemap.

the upcoming #13704 will resolve this (we will only look at real scripts)